test/lib/Launcher.ts

```diff
@@ -43,16 +46,21 @@ export class Launcher {
         return `http://localhost:${port}`;
     }
 
-    public async launch(count: number): Promise<ConnectorClient[]> {
-        const clients: ConnectorClient[] = [];
+    public async launch(count: number): Promise<ConnectorWithMetadata[]> {
+        const clients: ConnectorWithMetadata[] = [];
         const ports: number[] = [];
 
         for (let i = 0; i < count; i++) {
             const port = await getPort();
-            clients.push(ConnectorClient.create({ baseUrl: `http://localhost:${port}`, apiKey: this.apiKey }));
+            const client = ConnectorClient.create({ baseUrl: `http://localhost:${port}`, apiKey: this.apiKey });
+
+            clients.push(client);
             ports.push(port);
 
             const accountName = this.randomString();
+
+            clients[i]["_metadata"] = { accountName: `acc-${accountName}` };
```

Comment:
if you move the account name up you can directly assign the metadata to the client

Reply:
Sorry i dont get what you mean here

Reply:
see my other comment